packages/bp/src/core/users/chat-users/chat-users-repository.ts

```diff
@@ -141,7 +141,7 @@ export class ChannelUserRepository {
       .then(res => {
         return {
           id: res.id,
-          attributes: res.attributes,
+          attributes: this.database.knex.json.get(res.attributes),
```

Comment:
This didn't work before?

Reply:
I don't understand how it could have worked before as with SQLite, this function needs to be called in order for the JSON to be parsed!